commands/system/help.js

```diff
@@ -9,74 +11,170 @@ class Help extends Command {
       for (const c of sorted.values()) {
         if (c.help.category.toLowerCase() === type.toLowerCase()) {
-          if (c.help.category === "Owner" && level < 10 ) return;
           if (c.help.category === "NSFW" && !message.channel.nsfw) return;
           if (num < perpage * pg && num > perpage * pg - (perpage + 1)) {
             if (level < this.client.levelCache[c.conf.permLevel]) return;
-            output += `\n\`${message.settings.prefix + c.help.name}\` | ${c.help.description.length > 50 ? c.help.description.slice(0,50) +"...": c.help.description}`;
+            output += `\n\`${message.settings.prefix + c.help.name}\` | ${c.help.description.length > 80 ? `${c.help.description.slice(0, 80)}...` : c.help.description}`;
           }
           num++;
         }
       }
-    
+
       if (num) {
-        embed.setTitle("Command category help")
-          .setDescription(`A list of commands in the ${type} category.\n(Total of ${num} commands in this category)\n\nTo get help on a specific command do \`${message.settings.prefix}help <command>\`\n\n${num > 10 && pg === 1 ? `To view more commands do\` ${message.settings.prefix}help <category> 2\`` : "" }`)
-          .addField("Commands", output);
+        Math.ceil(num / perpage) === 1 ?
+          embed.setTitle(type.toProperCase())
+            .setDescription(`A list of commands in the ${type} category.\n(Total of ${num} commands in this category)\n\nTo get help on a specific command do \`${message.settings.prefix}help <command>\``)
+            .addField("Commands", output)
+          :
+          embed.setTitle(`Page ${page}/${Math.ceil(num / perpage)} for ${type.toProperCase()}`)
+            .setDescription(`A list of commands in the ${type} category.\n(Total of ${num} commands in this category)\n\nTo get help on a specific command do \`${message.settings.prefix}help <command>\``)
+            .addField("Commands", output);
       }
     }
 
-    if (this.client.commands.has(type) || this.client.commands.some(command => command.conf.aliases.includes(type))) {
-      const cm = this.client.commands.get(type) || this.client.commands.get(this.client.aliases.get(type));
+    if (this.client.commands.has(type) || this.client.aliases.has(type)) {
+      const cm = this.client.commands.get(type) || this.client.commands.get(this.client.aliases.get((type)));
       if (level < this.client.levelCache[cm.conf.permLevel]) return;
-      embed.setTitle(cm.help.name.toProperCase())
+      embed.setTitle(cm.help.name)
         .addField("Command description", cm.help.description)
         .addField("Command usage", `\`${cm.help.usage}\``)
-        .addField("Command aliases", cm.conf.aliases.length == 0 ? "None" : cm.conf.aliases.join(", ") );
+        .addField("Command aliases", cm.conf.aliases.length === 0 ? "None" : cm.conf.aliases.join(", "));
+    }
+    const msg2 = await message.channel.send(embed);
```

Comment:
This is an undescriptive name. Please rename this constant to another that represents better what it does.